core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestModule.kt

```diff
@@ -80,5 +80,9 @@ class RestModule(private val bindRemote : Boolean = true) : AbstractModule(){
 
         bind(HarvestActualHttpWsResponseHandler::class.java)
             .asEagerSingleton()
+
+        bind(SecurityRest::class.java)
+            .asEagerSingleton()
+
```

Comment:
bind(SecurityRest::class.java)
            .asEagerSingleton()

These two lines appear 3 times in different classes. I guess code duplication is inevitable in this case.